### ISSUE TITLE
In blender 2.83.2 gives error in console - and the textures are not properly set.

### ISSUE THREAD

**Issue by HauptmannTod** [a1studmuffin/SpaceshipGenerator#37]
Hello. 
The errors it gives in console are 
"Fake_module: addon missing bl_info gives bad performance!: _then it follows the scripts directory_ \\addons\\build.py"
"Fake_module: addon missing bl_info gives bad performance!: _then it follows the scripts directory_ \\addons\\spaceship_generator.py"
And when you add ship to the scene it is messed up and textures are not set properly.
Best Regards.

**Comment by ldo** [comment]
See #35.

**Comment by HauptmannTod** [comment]
Yes. I tried to install 1.6.0 and 1.6.2 - directly cloned and downloaded as zip and also installed from releases section. In 2.83.1 all worked properly, now it seems mess as windows are on random places for example. And I have not unpacked the zips. Do not know if I am missing something.

**Comment by ldo** [comment]
Maybe go into your `~/.config/blender/`*version*`/scripts/addons/` folder and get rid of all traces of any existing `add_mesh_SpaceshipGenerator` folder and contents, before trying the installation again.

**Comment by a1studmuffin** [comment]
Although the master branch was updated to support Blender 2.8.0, I only just got around to updating a new release ZIP (hooray for pandemics!). Give the v1.1.3 zip a go: https://github.com/a1studmuffin/SpaceshipGenerator/releases/tag/v1.1.3
